nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/services/azure/data/explorer/StandardKustoQueryService.java

```diff
@@ -159,10 +162,7 @@ private ConnectionStringBuilder getConnectionStringBuilder(final ConfigurationCo
             builder = ConnectionStringBuilder.createWithAadApplicationCredentials(clusterUrl, clientId, applicationKey, tenantId);
         }
 
-        final String vendor = System.getProperty("java.vendor");
-        final String version = System.getProperty("java.version");
-
-        builder.setConnectorDetails(vendor, version, null, null, false, null);
+        builder.setConnectorDetails("Kusto.Nifi.Source", StandardKustoQueryService.class.getPackage().getImplementationVersion(), null, null, false, null, NIFI_SOURCE);
```

Comment:
Yes this change is necessary since connector details/identifier along with version information is required for identification of request in ADX(Azure Data Explorer) cluster for debugging and tracking purpose. This is a standard across all the connectors we have implemented for Azure Data Explorer.